applications/exporter/application_round_result_exporter.py

```diff
@@ -0,0 +1,194 @@
+import csv
+from collections.abc import Iterable
+from dataclasses import asdict, dataclass
+from io import StringIO
+
+from django.db.models import Prefetch
+from django.http import FileResponse
+from lookup_property import L
+
+from applications.choices import ApplicationStatusChoice
+from applications.exporter.base_exporter import BaseCSVExporter, BaseExportRow
+from applications.models import AllocatedTimeSlot, ApplicationSection, ReservationUnitOption
+from applications.querysets.application_section import ApplicationSectionQuerySet
+from common.date_utils import local_date, local_date_string, local_time_string, local_timedelta_string
+
+
+@dataclass
+class ApplicationSectionExportRow(BaseExportRow):
+    # Application
+    application_id: str = ""
+    application_status: str = ""
+    applicant: str = ""
+
+    # Application Section
+    section_id: str = ""
+    section_status: str = ""
+    section_name: str = ""
+    reservations_begin_date: str = ""
+    reservations_end_date: str = ""
+    applied_reservations_per_week: str = ""
+    reservation_min_duration: str = ""
+    reservation_max_duration: str = ""
+
+    # Reservation Unit
+    reservation_unit_name: str = ""
+    unit_name: str = ""
+    day_of_the_week: str = ""
+    begin_time: str = ""
+    end_time: str = ""
+
+    # Price
+    price: str = ""  # TODO
```

Comment:
TODO?

Reply:
Updated the comment to be more descriptive